src/components/InventoryGroups/Modals/AddHostToGroupModal.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import validatorTypes from '@data-driven-forms/react-form-renderer/validator-types';
+import componentTypes from '@data-driven-forms/react-form-renderer/component-types';
+import Modal from './Modal';
+import { addHostToGroup } from '../utils/api';
+import apiWithToast from '../utils/apiWithToast';
+import { useDispatch } from 'react-redux';
+import { Button, Text } from '@patternfly/react-core';
+import SearchInput from './SearchInput';
+
+const CreateGroupButton = ({ closeModal }) => (
+    <>
+        <Text>Or</Text>
```

Comment:
I would better move this out from the component

Reply:
Why? Where should I move it? It's not used anywhere else...